stl/inc/xutility

```diff
@@ -644,13 +644,6 @@ struct iterator_traits<_Ty*> {
 
 // CUSTOMIZATION POINT OBJECT ranges::iter_move
 namespace ranges {
-    // STRUCT TEMPLATE _Choice_t

```

Comment:
How does this gets included back into `<xutility>`

Should there be an explicit include of `<compare>`?

Reply:
Resolving duplicate comment.